app/__init__.py

```diff
@@ -1,8 +1,10 @@
+import pytest
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
 from flask_migrate import Migrate
 import os
 from dotenv import load_dotenv
+import datetime
```

Comment:
since you aren't using datetime in this file, we don't need to import it. also, are you calling load_dotenv anywhere? were your environment variables able to load without it?

```suggestion
from dotenv import load_dotenv
load_dotenv()
```